projects/1.18/assets/female-gender-forge/wildfire_gender/lang/zh_cn.json

```
@@ -0,0 +1,62 @@
+{
+	"category.wildfire_gender.generic": "Wildfire 的性别模组",
+	"key.wildfire_gender.gender_menu": "性别信息调整菜单",
+
+	"wildfire_gender.hurt.female": "女性玩家受伤",
+
+	"wildfire_gender.player_list.title": "Wildfire 的性别模组",
+	"wildfire_gender.player_list.settings_button": "设置",
+	"wildfire_gender.player_list.sync_status": "同步状态",
+	"wildfire_gender.player_list.state.loading": "加载数据...",
+	"wildfire_gender.player_list.state.synced": "已经同步玩家的信息",
+
+	"wildfire_gender.wardrobe.title": "玩家特征设置",
+	"wildfire_gender.wardrobe.slider.breast_size": "胸部大小: %s%%",
+	"wildfire_gender.wardrobe.slider.separation": "胸部的里外: %s",

```

```suggestion
	"wildfire_gender.wardrobe.slider.separation": "乳房间距：%s",
```
```suggestion
	"wildfire_gender.wardrobe.slider.separation": "胸部间距：%s",
```
首先，请使用汉语标点
其次，请注意准确描述